examples/simple-examples/src/conversation/app/update.ts

```diff
@@ -0,0 +1,47 @@
+import { UpdateAppRequestData } from '@sinch/sdk-core';
+import {
+  getAppIdFromConfig, getMessengerTokenFormConfig,
+  getPrintFormat,
+  initClient,
+  printFullResponse,
+} from '../../config';
+
+(async () => {
+  console.log('*****************');
+  console.log('* App_UpdateApp *');
+  console.log('*****************');
+
+  const appId = getAppIdFromConfig();
+
+  const requestData: UpdateAppRequestData = {
+    app_id: appId,
+    update_mask: ['display_name', 'conversation_metadata_report_view'],
```

Comment:
See comment related to SDK usage simplification (https://github.com/sinch/sinch-sdk-node/pull/13#discussion_r1469240518)